src/past-purchase/index.ts

```diff
@@ -0,0 +1,24 @@
+import { alias, tag, Events, Selectors, Store, Tag } from '@storefront/core';
+
+@alias('saytCategories')
+@tag('gb-sayt-past-purchase', require('./index.html'))
+class PastPurchase {
+  $pastPurchase: Store.Autocomplete.Suggestion;
+
+  state: PastPurchase.State = {
+    onClick: (event: MouseEvent) => {
+      // todo:
```

Comment:
work pending?

Reply:
Yes, the routing wasn't done at this time, that's part of the next block of work on BBD-833. Would you prefer I remove it entirely in this ticket?